src/lib/common/resource_mgr.cpp

```diff
@@ -46,7 +48,21 @@ void ResourceMgr::stop() {
 //    writes on this descriptor;
 //
 void ResourceMgr::trigger_truncate() {
+    if (m_is_stopped_.load()) {
```

Comment:
NIT: use memory_order_release for line 35 and memory_order_acquire for this line.

using there two memory order , we can 100% guarantee if line 51 can be aware of the change of 
`m_is_stopped_`

Reply:
the default behavior of load() and operator= is equivalent of std::memory_order_seq_cst garunteens.